testing/solidity/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Trevis Schiffer <nikolaibitinit@gmail.com>
+# Maintainer: Trevis Schiffer <nikolaibitinit@gmail.com>
+
+pkgname=solidity
+pkgver=0.5.10
+pkgrel=1
+pkgdesc="The Solidity Contract-Oriented Programming Language"
+url="https://github.com/ethereum/solidity"
+arch="all"
+license="GPL3"
+depends=""
+makedepends="boost-dev cmake git"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/ethereum/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!check" # Test suite issues
+
+prepare() { 
+	cd $builddir 
+	git init
+	echo -n "" > commit_hash.txt
+	# Fix for GCC 8 
+	sed -e '/add_compile_options(-Wall)/d' -i.bak cmake/EthCompilerSettings.cmake
+}
+
+build() { 
+	cd "$builddir" 
+	mkdir -p build 
+	cd build 
```

Comment:
creating the build directory for out-of-tree builds usually happens in prepare()